common/saber_base.h

```diff
@@ -82,7 +82,7 @@ class SaberBase {
   // 1.0 = kDefaultVolume
   // This is really just for sound fonts.
   virtual void SetHumVolume(float volume) {}
-  virtual void StartSwing() {}
+  virtual void StartSwing(const Vec3& gyro, float swingThreshold, float slashThreshold) {}
```

Comment:
Why are swingThreshold and slashThreshold arguments?

Reply:
Because they are sourced from smoothsw.ini for smooth swing and config.ini for legacy fonts.